definitions/npm/lodash_v4.x.x/flow_v0.55.x-/lodash_v4.x.x.js

```diff
@@ -0,0 +1,1955 @@
+      updater: Function,
+      customizer?: Function
+    ): Object,
+    values(object?: ?Object): Array<any>,
+    valuesIn(object?: ?Object): Array<any>,
+
+    // Seq
+    // harder to read, but this is _()
+    (value: any): any,
+    chain<T>(value: T): any,
+    tap<T>(value: T, interceptor: (value: T) => any): T,
+    thru<T1, T2>(value: T1, interceptor: (value: T1) => T2): T2,
+    // TODO: _.prototype.*
+
+    // String
+    camelCase(string?: ?string): string,
+    capitalize(string?: string): string,
+    deburr(string?: string): string,
+    endsWith(string?: string, target?: string, position?: number): boolean,
+    escape(string?: string): string,
+    escapeRegExp(string?: string): string,
+    kebabCase(string?: string): string,
+    lowerCase(string?: string): string,
+    lowerFirst(string?: string): string,
+    pad(string?: string, length?: number, chars?: string): string,
+    padEnd(string?: string, length?: number, chars?: string): string,
+    padStart(string?: string, length?: number, chars?: string): string,
+    parseInt(string: string, radix?: number): number,
+    repeat(string?: string, n?: number): string,
+    replace(
+      string?: string,
+      pattern: RegExp | string,
+      replacement: ((string: string) => string) | string
+    ): string,
+    snakeCase(string?: string): string,
+    split(
+      string?: string,
+      separator: RegExp | string,
+      limit?: number
+    ): Array<string>,
+    startCase(string?: string): string,
+    startsWith(string?: string, target?: string, position?: number): boolean,
+    template(string?: string, options?: TemplateSettings): Function,
+    toLower(string?: string): string,
+    toUpper(string?: string): string,
+    trim(string?: string, chars?: string): string,
+    trimEnd(string?: string, chars?: string): string,
+    trimStart(string?: string, chars?: string): string,
+    truncate(string?: string, options?: TruncateOptions): string,
+    unescape(string?: string): string,
+    upperCase(string?: string): string,
+    upperFirst(string?: string): string,
+    words(string?: string, pattern?: RegExp | string): Array<string>,
+
+    // Util
+    attempt(func: Function, ...args: Array<any>): any,
+    bindAll(object?: ?Object, methodNames: Array<string>): Object,
+    bindAll(object?: ?Object, ...methodNames: Array<string>): Object,
+    cond(pairs: NestedArray<Function>): Function,
+    conforms(source: Object): Function,
+    constant<T>(value: T): () => T,
+    defaultTo<T1: string | boolean | Object, T2>(
+      value: T1,
+      defaultValue: T2
+    ): T1,
+    // NaN is a number instead of its own type, otherwise it would behave like null/void
+    defaultTo<T1: number, T2>(value: T1, defaultValue: T2): T1 | T2,
+    defaultTo<T1: void | null, T2>(value: T1, defaultValue: T2): T2,
+    flow: $ComposeReverse,
+    flow(funcs?: Array<Function>): Function,
+    flowRight: $Compose,
+    flowRight(funcs?: Array<Function>): Function,
```

Comment:
Why this still defined?

Reply:
Does $Compose work for both `(...fn)` and `(fn[])`?

Reply:
Ah, okay. Just confused a bit :)